app/jobs/process_patient_changesets_job.rb

```diff
@@ -51,7 +51,7 @@ def perform(patient_changeset, step_name = nil)
   private
 
   def unique_nhs_numbers(patient_changeset)
-    patient_changeset.search_results.pluck(:nhs_number).compact.uniq
+    patient_changeset.search_results.map { |sr| sr["nhs_number"] }.compact.uniq
```

Comment:
```suggestion
    patient_changeset.search_results.map { it["nhs_number"] }.compact.uniq
```

I don't think `sr` is any clearer than `it` 😄